app/helpers/application_helper/toolbar/middleware_datasource_center.rb

```diff
@@ -45,7 +45,8 @@ class ApplicationHelper::Toolbar::MiddlewareDatasourceCenter < ApplicationHelper
             'pficon pficon-delete fa-lg',
             N_('Remove Middleware Datasource'),
             N_('Remove'),
-            :confirm => N_('Do you want to remove this datasource ?'))
```

Comment:
it's not your code, but could you please do
datasource ? -> datasource?
(it's here and on other lines as well)